test/fixtures/shared/variables.tf

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+  * Copyright 2018 Google LLC
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *      http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+variable "admin_user" {
+  description = "The GSuite Admin user to impersonate"
```

Comment:
The intent of the new test framework is that every variable input is provided in `source.sh`. Additionally, we should try to minimize the number of variable values required to execute fixtures; the majority of inputs to the examples should be hard-coded in the fixtures (or better yet, the examples themselves should be more hard-coded).

Reply:
Do I have to create `source.sh` ? Can we split this in a separate issue and I'll take care of hardcoding examples after we merge this ?

Reply:
Opened #11

Reply:
`source.sh` is created by `make_source.sh`.